stdlib/public/Concurrency/Task.swift

```diff
@@ -438,23 +472,37 @@ public func detach<T>(
 }
 
 @discardableResult
-@_alwaysEmitIntoClient
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 public func asyncDetached<T>(
-  priority: Task.Priority = .unspecified,
+  priority: Task.Priority? = nil,
   @_implicitSelfCapture operation: __owned @Sendable @escaping () async -> T
 ) -> Task.Handle<T, Never> {
-  return detach(priority: priority, operation: operation)
+  return detach(priority: priority ?? .unspecified, operation: operation)
 }
 
 @discardableResult
-@_alwaysEmitIntoClient
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 public func asyncDetached<T>(
-  priority: Task.Priority = .unspecified,
+  priority: Task.Priority? = nil,
   @_implicitSelfCapture operation: __owned @Sendable @escaping () async throws -> T
 ) -> Task.Handle<T, Error> {
-  return detach(priority: priority, operation: operation)
+  return detach(priority: priority ?? .unspecified, operation: operation)
+}
+
+/// ABI stub while we stage in the new signatures
+@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
+@usableFromInline
+func async(
+  priority: Task.Priority,
+  @_inheritActorContext @_implicitSelfCapture operation: __owned @Sendable @escaping () async -> Void
+) {
+  let adjustedPriority: Task.Priority?
+  if priority == .unspecified {
+    adjustedPriority = nil
+  } else {
+    adjustedPriority = priority
+  }
+  let _: Task.Handle = async(priority: adjustedPriority, operation: operation)
```

Comment:
Huh, without `<>`?

Reply:
Type inference is MAGIC

Reply:
are you serious? LOL That's pretty both fun / hilarious / terrifying at the same time 🤣 

Interesting, did not know about that... Still tained by my scala days I guess, one had to write <_, _>